Makefile

```diff
@@ -111,6 +112,30 @@ test-sagemaker: build-tests
 	--region ${REGION} \
 	--domain ${AWS_DOMAIN}
 
+
+test-sagemaker-history-server: build-tests
+	pipenv run pytest --reruns 3 -s -vv test/integration/history \
+	--repo=$(DEST_REPO) --tag=$(VERSION) --durations=0 \
+	--spark-version=$(SPARK_VERSION) \
+	--framework-version=$(FRAMEWORK_VERSION) \
+	--role $(ROLE) \
+	--image_uri $(IMAGE_URI) \
+	--region ${REGION} \
+	--domain ${AWS_DOMAIN}
+
+
+test-sagemaker-processing: build-tests
```

Comment:
Adding a separate test for easier and faster iteration of integ tests. The original target `test-sagemaker` remains